### PR TITLE
Why FBG post

### PR DESCRIPTION
**Pull request by flamecoals**

### CODE REVIEW

File: blog/source/_posts/WhyFbg.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+tags: why motivation boardgame.io freeboardgame.org
+---
+
+One of the main pain points I experienced trying to play board games online with friends are all the fragmented implementations out there. Some amazing games only work for iOS, or others only for Android, or some only in Desktop. Most of these versions don't have multiplayer modes that can be played with each other, some are extremely expensive, and most require their own account. Besides, having one friend with the "wrong" OS will make it impossible for the party of friends to play together.
+
+So if there are so many board game implementations out there, how would someone in their sane mind think that [yet another board game project would help ?](https://xkcd.com/927/) 
+
+![XKCD](https://imgs.xkcd.com/comics/standards.png)
+
+Our bet is that FreeBoardGame.org is different: While most implementations out there are closed source, we are a Free software project. That means that instead of competing with each other in creating yet another game, we are going to slowly but surely share our work and create games that can be played, modified, improved, studied or even forked by anyone.
```
Comment (jasonharrison) [inline blog/source/_posts/WhyFbg.md]: "Free" in "Free software project" should be lowercase

File: blog/source/_posts/WhyFbg.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+With that said, we are also not the first board game FOSS project. Even though there are several FOSS board game projects out there, most of them are one-off projects that implement one single game. An exception is [boardgame.io](https://boardgame.io), a framework that makes it easy to code turn based games. This project created a large community around it, and FreeBoardGame.org is built on top of it.
```
Comment (jasonharrison) [inline blog/source/_posts/WhyFbg.md]: Most people do not know what FOSS means

File: blog/source/_posts/WhyFbg.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+In fact, in more practical terms, you can think FreeBoardGame.org being a publishing platform for your boardgame.io games. While implementing the game requires a good amount of work, so does delivering a good experience for thousands of users, and this will be our focus. This includes helping with the development and curating games that get into the platform, fostering the community of users, making sure that the website is fast and optimized, making sure it is discoverable, etc.
+
+Being a free (as in freedom) software project does not only mean that we will provide the source code to anyone, but also that we will have an open governance. Hence, we will always make sure to give credits where credits are due (i.e. credits for all games developed by a contributor). And if we ever make any revenue from the website, these funds will be spent as transparently as possible: prioritizing maintaining the servers, investing in developing more games, improving current games, developing the infrastructure further etc. This way, we increase our odds of having a sustainable project and you can be sure that all your work will go towards the community.
```
Comment (jasonharrison) [inline blog/source/_posts/WhyFbg.md]: Open governance?  I'm not sure if this is the best term to use.

File: blog/source/_posts/WhyFbg.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+Therefore, together we can join all these one-off efforts into a single, coherent community effort. You can have fun implementing some game you really like, or you can help building the infra-structure for several games at once. Either way, your effort will be multiplied by the effort of others in the community, as your game and contributions can reach thousands of users easily. Let's have fun together :).
```
Comment (jasonharrison) [inline blog/source/_posts/WhyFbg.md]: I think infrastructure is one word. I would also change "some game" to "a game".  "Your effort" to "Your efforts".

File: blog/source/_posts/WhyFbg.md (hunk 1 above)
Comment (flamecoals) [reply blog/source/_posts/WhyFbg.md]: You are technically correct, but this is for emphasis

File: blog/source/_posts/WhyFbg.md (hunk 2 above)
Comment (flamecoals) [reply blog/source/_posts/WhyFbg.md]: Our target audience does

File: blog/source/_posts/WhyFbg.md (hunk 3 above)
Comment (flamecoals) [reply blog/source/_posts/WhyFbg.md]: Thats the correct term

File: blog/source/_posts/WhyFbg.md (hunk 4 above)
Comment (flamecoals) [reply blog/source/_posts/WhyFbg.md]: Done.

**Pull request merged**